CHANGELOG.md

```diff
@@ -4,8 +4,8 @@
 - []  ([#](https://github.com/airbnb/react-dates/pull/))
 -->
 
-## [Unreleased]
-- [fix] Fix date selection in the SDP ([#1530])
+## 20.1.1
```

Comment:
please revert this line; version numbers are determined only at release time.
```suggestion
## [Unreleased]
```

Reply:
reverted